packages/api-generator/src/locale/en/v-table.json

```diff
@@ -3,6 +3,6 @@
     "fixedHeader": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "fixedFooter": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "height": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
-    "hover": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))"
+    "hover": "If hover is true it will add a transition effect to a table's row when the mouse is over it."
```

Comment:
I think transition is the wrong word to use here. It should simply say we apply a hover effect, or a different background

Reply:
Hi, yes it makes sense. Changed the word in the next commit.